src/bot/integrations/discord.ts

```diff
@@ -98,6 +98,28 @@ class Discord extends Integration {
   })
   sendGeneralAnnounceToChannel = '';
 
+  @settings('bot')
+  @ui({
+    type: 'selector', values: ['online', 'idle', 'invisible', 'dnd'],
+    if: () => self.guild.length > 0,
+  })
+  onlinePresence: 'online' | 'idle' | 'invisible' | 'dnd' = 'online';
+
+  /*
```

Comment:
this is commented

Reply:
i know i remove it in next commit also have regrouped the status to its own label